src/schema/ecommerce/orders.js

```diff
@@ -13,14 +14,14 @@ export const Orders = {
   name: "Orders",
   type: OrderConnection,
   description: "Returns list of orders",
-  args: {
+  args: pageable({
```

Comment:
So these will make Metaphysics accept these args, but do they get used/passed to Exchange below?

Reply:
yeah good point, we need to add it to list of params 👇